examples/platforms/efr32/sleepy-demo/README.md

```diff
@@ -0,0 +1,98 @@
+
+# EFR32 Sleepy Demo Example
+
+The EFR32 Sleepy applications demonstrates Sleepy End Device behaviour using
+the EFR32's low power EM2 mode. The steps below will take you through the
+process of building and running the demo
+
+For setting up the build environment refer to [examples/platforms/efr32/README.md](../../platforms/efr32/README.md).
+
+
+## 1. Build
+
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+$ make -f examples/Makefile-efr32 COMMISSIONER=1 JOINER=1 DHCP6_CLIENT=1 DHCP6_SERVER=1 BOARD=BRD4161A
+```
+
+Convert the resulting executables into S-Record format and append a s37 suffix.
```

Comment:
Can we reference the "Flash Binaries" section in `examples/platforms/efr32/README.md`?